_includes/topnav.html

```diff
@@ -14,98 +14,49 @@
-                        <span class="caret"></span>
-                    </a>
-                    <ul class="dropdown-menu">
-                        <li>
-                            <a href={{ "/docs/tutorial/deployment"  | relative_url }}>Deployment</a>
-                        </li>
-                        <li class="divider"></li>
-                        <li>
-                            <a href={{ "/docs/tutorial/customization"  | relative_url }}>Customization</a>
-                        </li>
-                    </ul>
-                </li>
-                <li>
-                    <a class="dropdown-toggle" data-toggle="dropdown" href="#" aria-expanded="false">
-                      Datasets
+                      Benchmark
                       <span class="caret"></span>
                     </a>
                     <ul class="dropdown-menu">
                         <li>
-                            <a href={{ "/docs/dataset"  | relative_url }}>Overview</a>
-                        </li>
-                        <li class="divider"></li>
-                        <li>
-                            <a href={{ "/docs/dataset/cv"  | relative_url }}>CV Dataset</a>
+                            <a href={{ "/docs/dataset"  | relative_url }}>FedScale Datasets</a>
                         </li>
                         <li class="divider"></li>
                         <li>
-                            <a href={{ "/docs/dataset/nlp"  | relative_url }}>NLP Dataset</a>
+                            <a href={{ "/docs/leader_overview"  | relative_url }}>Leaderboard Rules</a>
                         </li>
                         <li class="divider"></li>
                         <li>
-                            <a href={{ "/docs/dataset/misc"  | relative_url }}>Misc Dataset</a>
-                        </li>
-                        <li class="divider"></li>
-                        <li>
-                            <a href={{ "/docs/dataset/sys"  | relative_url }}>System Traces</a>
-                        </li>
-                    </ul>
-                </li>
-                <li>
-                    <a class="dropdown-toggle" data-toggle="dropdown" href="#" aria-expanded="false">
-                      APIs
-                      <span class="caret"></span>
-                    </a>
-                    <ul class="dropdown-menu">
-                        <li>
-                            <a href=https://fedscale.readthedocs.io/en/latest>Overview</a>
-                        </li>
-                        <li class="divider"></li>
-                        <li>
-                            <a href=https://fedscale.readthedocs.io/en/latest/fedscale.core.aggregation.html#module-fedscale.core.aggregation.aggregator>Aggregator</a>
+                            <a href={{ "/leaderboard/aggregation.html"  | relative_url }}>Aggregation Optimizations</a>
                         </li>
-                        <li class="divider"></li>
```

Comment:
Are you removing these "divider"s intentionally? It does not look good to me, as texts are crowded.

Reply:
Dividers between every item has little value. There are dividers between groups of related items